src/utils/__mocks__/toast.js

```diff
@@ -0,0 +1 @@
+export default jest.fn(() => {})
```

Comment:
Is the code that executes when this statement runs in the test? `jest.mock("../../utils/toast")`

Reply:
it's used in ```src/views/Settings/index.test.js``` to check if ```Toast``` was called. Basically it mocks ```Toast``` calls.  I intended to use it to test other ```Toast``` calls in other tests.

Reply:
I don't think this file is needed. AFAIK this file sets up a Manual mock that can be used to return data when certain calls are made against it. However, we are only concerned if Toast has been called with certain parameters. So I think converting this to an automock (by deleting this file) should still suit the needs. Let me know if I'm misunderstanding anything as I'm very new to Javascript testing.

Reply:
I'm really new to JS and JS testing too. I'll try the automock and see what happens.